### ISSUE TITLE
Review new facebook login flow and open graph - possibly deprecated by Dec 25th

### ISSUE THREAD

**Issue by brendanheywood** [theCrag/website#1613]
I got an email about this from FB, not sure if it even applies to us but need to check:

https://developers.facebook.com/docs/apps/upgrading

**Comment by scd** [comment]
I have just submitted our app for review. Wow Facebook has confusing process. After some investigation I worked out that we now have to apply to use the publish_actions.

**Comment by brendanheywood** [comment]
Jed pinged me and said that the fb publishing wasn't working, then realised it wasn't turned on, but it previously was on. I was wondering whether this change reset everyone's link and they need to reconnect and whether we should let everyone know?

**Comment by scd** [comment]
Did it start working for him again after turning it on? I must confess, like most facebook procedures, confusing reigns and I am not sure whether we did everything we had to. The main thing was getting authorization for our app to ask for specific permissions - which we did. I imagine that this process would ask for people to ok it again. If Jed's fb posts are now working then this seems like a plausible explanation so we should put a notice up on facebook. Can I leave it with you.
